.travis.yml

```diff
@@ -1,46 +0,0 @@
----
-rvm:
-  - 2.5
-cache: bundler
-sudo: required
-group: edge
-
-services: docker
-
-install: bundle install
-bundler_args: "--binstubs=$PWD/bin --jobs 3 --retry 3 --without development"
-
-branches:
-  only:
-    - master
-
-env:
-  global:
-    - KITCHEN_YAML=kitchen.yml
-    - INSTANCE=default-fedora
-    - CHEF_LICENSE="accept-no-persist"
```

Comment:
This needs to get moved into the action env. Also dokken needs to support setting this the right way. That's a huge bug.

Reply:
Yeah, because it seems like we don't have a standard test harness, or possibly a standard spec, we have lots of plugins that don't accept what they should.